### ISSUE TITLE
iPad display is messed up after the new scrolling feature.

### ISSUE THREAD

**Issue by levelonedev** [neagle/gokibitz#74]

**Comment by levelonedev** [comment]
![image](https://cloud.githubusercontent.com/assets/9813893/7475419/b8ead97c-f304-11e4-893c-a49f9ff947b7.jpg)

**Comment by neagle** [comment]
I meant to check last night, but I ended up focused on getting password changing working. If you happen to get a chance, could you see if this is fixed by cd0ccc57bd7d6d92a12e2138faf6dc0617953e35?

**Comment by levelonedev** [comment]
I will look at it tonight. Is the fix live now?

**Comment by neagle** [comment]
Yep. As I said, though, it's an "in theory" fix :)

--
Nate Eagle

**Comment by levelonedev** [comment]
Ok I will also look out for the bug @lukas1 found. #76

**Comment by lukas1** [comment]
I've checked with iPad mini, with the older ios 7.1.2 (I deliberately keep it on older version). First observation is, that I think the board is small on iPad... Because there's either game info on the left side of it, in portrait rotation, or comments AND game info in landscape orientation. In landscape orientation, if there are a lot of comments and you scroll down to see comments below the screen, the position of game info and comments stays fixed, however as below game info there's empty space, when you scroll down so that game info is no longer visible on screen, the board jumps to the left edge of the screen.

I'm not a fan of this layout on ipad (basically the same as in desktop browser) - the board is too small (the board is too small even in portrait orientation on iPad).

The layout on Nexus 7 -> 7inch android tablet is ok, in portrait mode, basically the same as mobile; in landscape mode, the board is small as game info is on the right side of the board(I think having comments there instead would be bit better), but I wouldn't care all that much about it, because people with nexus 7 will anyway use the tablet in portrait mode for the most part and if they don't like landscape layout of the website, portrait layout is ok. This is different from iPad, where in both portrait and landscape orientation the board is just small.

I've also tested with Nexus 10, 10inch android tablet, that users mostly use in landscape orientation; Although the layout doesn't experience the same kind of bugs as iPad, or my desktop browser, I think the board is small.
